_non-cosmos/ethereum/assetlist.json

```diff
@@ -2404,6 +2404,99 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/FBTC.png"
       }
+    },
+    {
+      "type_asset": "erc20",
+      "address": "0x69223B5B36a785Ec08e5f685fd7961399982C566",
+      "denom_units": [
+        {
+          "denom": "0x69223B5B36a785Ec08e5f685fd7961399982C566",
+          "exponent": 0
+        },
+        {
+          "denom": "satwbtc",
+          "exponent": 8
+        }
+      ],
+      "base": "0x69223B5B36a785Ec08e5f685fd7961399982C566",
+      "name": "Satlayer WBTC",
+      "display": "satwbtc",
+      "symbol": "satWBTC",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"
```

Comment:
This base reference is case-sensitive, and WBTC is registered as 0x2260fac5e5542a773aa44fbcfedf7c193bc2c599